marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command/Load.hs

```diff
@@ -1,333 +1,89 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE KindSignatures #-}
-{-# LANGUAGE ViewPatterns #-}
 
 module Language.Marlowe.Runtime.CLI.Command.Load (
   LoadCommand (..),
   loadCommandParser,
   runLoadCommand,
 ) where
 
-import Control.Monad (foldM, unless)
-import qualified Control.Monad as Monad
-import Control.Monad.Except (ExceptT (ExceptT), runExceptT, throwError, withExceptT)
-import Control.Monad.Reader (MonadReader (ask), ReaderT, local, runReaderT)
+import Control.Monad (forever, replicateM, unless)
```

Comment:
This file should be updated with the zip and manifest changes https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe-runtime-cli/README.md